### PR TITLE
Use relative line-height

### PR DESCRIPTION
**Pull request by dashouse**
- Converts line heights to relative unit-less value

This needs to be tested on Samsung Internet browser to see if it fixes issue brought up in Accessibility audit.

### CODE REVIEW

**Comment by nickcolley** [comment]
Thanks for this Dave!

Could we break these into a RFC for the rem stuff and a straightforward PR for the uncontentious stuff?

**Comment by hannalaakso** [comment]
I've tested this on the following and it looks good:
Chrome (latest) on Mac
Chrome (Samsung Android) with text resized to 200% in Chrome settings
Samsung Internet (Samsung Android) with text resized to "Huge" in phone settings
Firefox (latest) on Mac
Safari 11.1 on Mac
iOS 11 Safari with text resized to max size in phone settings
IE8-11 on Windows

To test, I made the examples in `examples/typography` longer in order to check text wrapping.

Just need a changelog entry for the PR.

**Comment by nickcolley** [comment]
Firefox before and after:

screen shot 2018-06-28 at 11 11 45
screen shot 2018-06-28 at 11 11 40

File: src/settings/_typography-responsive.scss [hunk 1]
```diff
@@ -26,11 +26,11 @@ $govuk-typography-scale: (
   80: (
     null: (
       font-size: 53px,
-      line-height: 55px
+      line-height: 1.0377358491 // 55px
```
Comment (36degrees) [inline src/settings/_typography-responsive.scss]: I do wonder if these would have been clearer if written as:

```scss
line-height: 55px / 53px;
```

Thoughts?

Comment (36degrees) [reply src/settings/_typography-responsive.scss]: Alternatively, we could have left these as pixel values in the font-map and calculated them in the typography-responsive mixin.

Comment (dashouse) [reply src/settings/_typography-responsive.scss]: I tried various ways of doing this calculation but kept running into issues...Should probably refactor so it's more "automatic" / less prone to typos. If we haven't released can we carry on?

Comment (nickcolley) [reply src/settings/_typography-responsive.scss]: If we don't expect this to change often, might be worth doing less here?

Comment (36degrees) [reply src/settings/_typography-responsive.scss]: Yep, happy with this. I've been playing locally with this too – looks like our font-size and line-height are ending up as strings somehow, which means using them in calculations doesn't work. It'd be great to be able to define everything in pixels and everything 'just work', but happy with this for now 👍

Comment (dashouse) [reply src/settings/_typography-responsive.scss]: I kind of got something interesting when spitting it out as a string if we change the output to `font: 16 / 20` and not define line height independently... but this didn't play very nicely with the other PR that needed two font-size declarations.

**Pull request merged**